tests/sles4sap/ase_test.pm

```diff
@@ -0,0 +1,85 @@
+# SUSE's SLES4SAP openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: SAP ASE smoke test. Run some commands and queries to verify SAP ASE is running
+# Requires: sles4sap/ase_install
+# Maintainer: QE-SAP <qe-sap@suse.de>
+
+use base "sles4sap";
+use strict;
+use warnings;
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+
+sub run_ase_query {
+    my %args = @_;
+    die "Got no query!" unless $args{query};
+    $args{timeout} //= 30;
+    assert_script_run "echo '$args{query}' > query.file; echo go >> query.file";
+    assert_script_run 'cat query.file';
```

Comment:
is this `cat` a debug only line?

Reply:
Yes, to show exactly what query was executed.

Reply:
I mean, is not debug for me as a developer, I'm intending it to be a debug for the reviewer.